step-capstone/src/components/Event.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Box, Grid, Typography } from '@material-ui/core'
+
+/*
+Takes data in this form:
+        {
+          finalized: true,
+          type: "event",
+          title: "Visiting Sherlock",
+          location: "221b Baker Street, London",
+          timeStart: "8:30",
```

Comment:
Timezones would be a good thing to add here, including display handling for them.